WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -165,6 +167,8 @@ public void ValidateInputRequest(ILogger log)
                 log.Error(INVALID_DATE, () => new { simulation = this });
                 throw new BadRequestException(INVALID_DATE, e);
             }
+
+            await IotHubConnectionStringManager.ValidateConnectionString(this.IotHub.ConnectionString);
```

Comment:
avoid static calls, inject the IotHubConnectionStringManager similarly to how the logger is passed in